### PR TITLE
Task/jest

### PR DESCRIPTION
**Pull request by esotrope**
image

this faster?

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -47,26 +50,18 @@
     "zone.js": "~0.9.1"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^0.800.6",
+    "@angular-devkit/build-angular": "^0.803.23",
```
Comment (esotrope) [inline webapp/package.json]: result of `npm audit fix`

File: webapp/package.json [hunk 2]
```diff
@@ -47,26 +50,18 @@
...
-    "karma": "~4.1.0",
-    "karma-chrome-launcher": "^2.2.0",
-    "karma-coverage-istanbul-reporter": "^2.1.0",
-    "karma-firefox-launcher": "^1.2.0",
-    "karma-jasmine": "~2.0.1",
-    "karma-jasmine-html-reporter": "^1.4.0",
-    "protractor": "~5.4.0",
-    "puppeteer": "^1.20.0",
+    "jest": "^24.9.0",
+    "jest-preset-angular": "^8.0.0",
```
Comment (esotrope) [inline webapp/package.json]: so many less deps

File: webapp/src/app/common/logging/logging.service.spec.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { TestBed } from '@angular/core/testing';
 import { LoggingService } from './logging.service';
-import { initializeSettingsProvider } from 'src/app/app.module.spec';
+import { initializeSettingsProvider } from 'src/app/mocks';
```
Comment (esotrope) [inline webapp/src/app/common/logging/logging.service.spec.ts]: jest had a problem with spec files without tests

**Comment by esotrope** [comment]
This could be the case if we have test code referenced in our source code? I'll take a look

**Comment by kb-smarterbalanced** [comment]
Closing because abandoned

**Pull request closed without merging**